### PR TITLE
Fix BitCarousel component style issues (#1584)

### LINKED ISSUE #1584: Modify BitCarousel component styles
The `BitCarousel` component styles have some issues and need to be modified.
In this photo, there are some points that could be implemented better and more user friendly:
freesnippingtool com_capture_20220306122615

### PR DESCRIPTION
**Pull request by saeedhalab**
This closes #1584

### CODE REVIEW

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Carousel/BitCarousel.Fluent.scss [hunk 1]
```diff
@@ -28,26 +28,34 @@
...
+        width: 8px;
+        margin: 0 4px;
+        background-color: #484644;
         border-radius: 50%;
         display: inline-block;
         transition: background-color 0.6s ease;
 
         &:hover, &.bit-crsl-dot-current {
-            background-color: #717171;
+            background-color: #001dff;
```
Comment (maryamhdr) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Carousel/BitCarousel.Fluent.scss:52]: plz use a color variable declared in _colors.fluent.scss

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Carousel/BitCarousel.Fluent.scss [hunk 2]
```diff
@@ -11,15 +11,15 @@
     & .bit-crsl-next-btn, .bit-crsl-previuse-btn {
         cursor: pointer;
         position: absolute;
-        top: 38%;
+        top: 45%;
         width: auto;
         margin-top: -22px;
-        padding: 11px 16px;
+        padding: 4px 0;
```
Comment (maryamhdr) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Carousel/BitCarousel.Fluent.scss:17]: plz convert all the values provided in px to rem

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Carousel/BitCarousel.Fluent.scss [hunk 3]
```diff
@@ -11,15 +11,15 @@
...
         color: white;
```
Comment (maryamhdr) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Carousel/BitCarousel.Fluent.scss:18]: use the $White variable instead

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Carousel/BitCarousel.Fluent.scss [hunk 4]
```diff
@@ -28,26 +28,34 @@
...
     & > div {
         cursor: pointer;
-        height: 15px;
-        width: 15px;
-        margin: 0 2px;
-        background-color: #bbb;
+        height: 8px;
+        width: 8px;
+        margin: 0 4px;
+        background-color: #484644;
```
Comment (maryamhdr) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Carousel/BitCarousel.Fluent.scss:46]: plz use a color variable declared in _colors.fluent.scss

**Review by maryamhdr — CHANGES_REQUESTED** [review]

**Review by maryamhdr — APPROVED** [review]

**Review by msynk — APPROVED** [review]

**Pull request merged**